<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Grouping-Suite]" guice-stage="DEVELOPMENT" verbose="6">
  <listeners>
    <listener class-name="utils.ReportUtil"/>
    <listener class-name="utils.RetryUtil"/>
  </listeners>
  <test thread-count="5" name="Test(failed)" verbose="6">
    <groups>
      <run>
        <include name="Master"/>
      </run>
    </groups>
    <classes>
      <class name="tests.LoginTest">
        <methods>
          <include name="verifyLoggingAfterchangingPassword_Test"/>
          <include name="verifyNavigatingBackAfterLogout_Test"/>
          <include name="verifyNavigatingBackAfterSuccessfullLogin_Test"/>
          <include name="tearDown"/>
          <include name="SetUp"/>
        </methods>
      </class> <!-- tests.LoginTest -->
      <class name="tests.RegisterTest">
        <methods>
          <include name="SetUp"/>
          <include name="verifyUserCannotuseInvalidPhoneNumber_Test" invocation-numbers="0 1 2 3 4"/>
          <include name="verifyPassWordFieldFollowComplexityStandards_Test" invocation-numbers="0 1 2 3 4 5 6"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- tests.RegisterTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Grouping-Suite] -->
